platform-ui-skin/src/main/webapp/skin/less/platform/portlets/extensions/login.less

```diff
@@ -73,6 +74,11 @@ input[type="checkbox"]{
     height: 17px!important;
     margin-right: 10px;
 }
+@media (max-width: 800px) {
+.brandingImageContent img.brandingImage {
+  display: none;
```

Comment:
The logo should hidden just in case the user is typing. Please check the invision link